soc/nxp/lpc/Kconfig

```diff
@@ -0,0 +1,19 @@
+# Copyright 2017,2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_FAMILY_LPC
+	select HAS_SEGGER_RTT if ZEPHYR_SEGGER_MODULE
+	select CLOCK_CONTROL
+	select ARM
+
+if SOC_FAMILY_LPC
+
+rsource "*/Kconfig"
+
+config BUILD_WITH_TFM
+	default y if TRUSTED_EXECUTION_NONSECURE
```

Comment:
Move to `Kconfig.defconfig` file

Reply:
Note: these issues do not appear to have been fixed

Reply:
will do.... I might have added this back in and was confused about where to put them